demos/demo_solo12.cpp

```diff
@@ -7,7 +7,7 @@
  * This file uses the Solo12 class in a small demo.
  */
 
-
+#include <numeric>
```

Comment:
Why not put it in common_programs_header.hpp?

Reply:
Hm, as the include is only needed in these few specific files, I think it's better to keep them there.